<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>51</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Imagenes/sin-contacto (2).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>321</y>
      <width>761</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>Acerque el dispositivo para realizar el cobro</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_tipo">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>370</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>Menor: $10.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>Cobrado: 1 $0.00
Por cobrar: 2 $10.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pagar_con_efectivo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: #28a745;  /* Verde tipo éxito */
        color: white;
        border: none;
        border-radius: 12px;
        padding: 12px 24px;
        font-size: 16px;
        font-weight: bold;
        letter-spacing: 1px;
    }
    QPushButton:hover {
        background-color: #218838;
    }
    QPushButton:pressed {
        background-color: #1e7e34;
    }
</string>
    </property>
    <property name="text">
     <string>Pago con efectivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancelar">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>410</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Aplica solo al botón con objectName &quot;btn_cancelar&quot; */
QPushButton#btn_cancelar {
    background-color: #dc3545;   /* Rojo tipo peligro */
    color: white;
    border: none;
    border-radius: 12px;
    padding: 12px 24px;
    font-size: 16px;
    font-weight: bold;
    letter-spacing: 1px;
}
QPushButton#btn_cancelar:hover {
    background-color: #c82333;
}
QPushButton#btn_cancelar:pressed {
    background-color: #bd2130;
}
</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
